code/game/turfs/open/_open.dm

```diff
@@ -15,6 +15,32 @@
 	var/clawfootstep = null
 	var/heavyfootstep = null
 
+	var/broken_icon = 'icons/turf/turf_damage.dmi'
+	var/broken = FALSE
+	var/burnt = FALSE
+
+	var/list/broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5", "damaged6", "damaged7")
+	var/list/broken_dirt_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5", "damaged6", "damaged7")
```

Comment:
do note that this is one of the MOST INSTANTIATED TYPES in the entire game.

Don't be making lists in its static constructor - for these make a global list and then link them instead, since the list isn't being modified.

For empty lists, just use a lazylist and have it start as null (for `damage_overlays`)